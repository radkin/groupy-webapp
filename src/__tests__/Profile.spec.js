import me from './data';

test('proper profile data is correct', () => {
  expect(me).toMatchSnapshot();
  expect(me.first).toEqual('jon');
});

test('me object should have properties id, first, last, zipCode, color,\
  initials, profileImage', () => {
    expect(me).toHaveProperty('id');
    expect(me).toHaveProperty('first');
    expect(me).toHaveProperty('last');
    expect(me).toHaveProperty('zipCode');
    expect(me).toHaveProperty('color');
    expect(me).toHaveProperty('initials');
    expect(me).toHaveProperty('profileImage');
  });
