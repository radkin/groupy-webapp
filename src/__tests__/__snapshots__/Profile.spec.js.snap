// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proper profile data is correct 1`] = `
Object {
  "color": "0xffff7cae",
  "first": "jon",
  "id": "5e62ade15014b544a50af943",
  "initials": "RR",
  "last": "smith",
  "profileImage": "https://avatars3.githubusercontent.com/u/467568?s=400&v=4",
  "zipCode": "00001",
}
`;
